### DISCUSSION TITLE
Is OME supporting multiple stream source input into one stream output

### DISCUSSION THREAD

**Discussion by billyriantono** [OvenMediaLabs/OvenMediaEngine#1467] [Help]
Hi,
Is anyone know is OME supporting multiple bitrate hls using multiple input source ?

for example : 
I have input source for 360p
I have input source for 720p

I want to make those two source to be one output for example streamA,
So later streamA have 2 resolution / multi bitrate.

Thank you

**Comment by getroot** [comment]
No, OME doesn't support it. Instead, you can create your own master m3u8 file and host it.

**Comment by getroot** [comment]
It has been released!

https://github.com/AirenSoft/OvenMediaEngine/discussions/1489
